Segment-Amplitude.podspec

```diff
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Pod/Classes/**/*'
 
-  s.dependency 'Analytics'
-  s.dependency 'Amplitude-iOS'
+  s.dependency 'Analytics', '~> 3.6' # support for iOS 7 +
```

Comment:
Can we actually change this back to the dependencies we are trying to bump to: ie: Analytics (3.7.0) and Amplitude-iOS (4.8.2)?